java/sage/media/sub/SRTSubtitleHandler.java

```diff
@@ -159,4 +444,67 @@ else if (!inTag)
     }
   }
 
+  @Override
+  public long getTimeTillUpdate(long currMediaTime)
+  {
+    long next = super.getTimeTillUpdate(currMediaTime);
+
+    if (monitorThread == null)
+      return next;
+
+    if (next == sage.Sage.MILLIS_PER_WEEK)
```

Comment:
I'd suggest changing this to be another constant such as NO_MORE_SUBS_LONG_WAIT and changing the other places its used in the subtitle classes to match that, also makes the interdependence more clear.